waitfor.go

```diff
@@ -154,9 +156,20 @@ func HTTPWaiter(name string, target *TargetConfig) error {
 	if err != nil {
 		return fmt.Errorf("could not connect to %s: %v", name, err)
 	}
-
-	if !isSuccess(resp.StatusCode) {
-		return fmt.Errorf("got %d from %s", resp.StatusCode, name)
+	if target.RegexStatus != "" {
```

Comment:
I think if we use the default of `200` in the parameter, we can remove the code paths that deal with an empty pattern. It'll also boost the code coverage.

Reply:
That's brilliant , I don't know why I didn't think about it !